tests/testthat/test-other-color-setting.R

```diff
@@ -57,7 +57,7 @@ test_that("boxplot ColorScheme", {
                            groupingFactors          = list("dose"),
                            smpTitleScaleFontFactor  = 1.8,
                            stringSampleFactors      = list("dose"),
-                           title                    = "Boxplot - ColorScheme: Set1"
+                           title                    = "Boxplot - ColorScheme: Set2"
```

Comment:
Any reason we changed this?

Reply:
while testing I found out title was wrong ,,
colorScheme was set to "Set2",